BUILDING_ON_WINDOWS.md

```diff
@@ -13,7 +15,37 @@ Notes:
 - This installation will take about 17 GB of disk space
```

Comment:
`21 GB` - I couldn't easily find the other ones in the VoD, but finding this one was easy due to the reaction

`#L3` might need an update too as temp storage is required to download Qt